class7-week3/models/todo.js

```diff
@@ -75,6 +75,60 @@ class Todo {
     })
   }
 
+  clear(callback) {
+    this.load((error, todos) => {
```

Comment:
This function is meant to remove all todos. Here you load the current todos, then you save them again in line 82. Does this accomplish what you want? (Hint: no, it doesn't)